app/api/resources/user.py

```diff
@@ -544,6 +544,6 @@ def get(cls):
         user_id = get_jwt_identity()
         dashboard = DAO.get_user_dashboard(user_id)
         if not dashboard:
-            return messages.USER_NOT_FOUND, HTTPStatus.NOT_FOUND
+            return messages.USER_NOT_FOUND, HTTPStatus.NOT_FOUND.value
```

Comment:
can you remove the `.value` from these instances where the enum is used in the code logic. This is issue is about changing it only on places where it will reflect on Swagger UI. This is also causing changes in the test coverage. Usually, when it is used in the context of an annotation like:
```
@users_ns.response(HTTPStatus.NOT_FOUND.value, "User not found")
```
So where there is a return statement involved, you can remove the `.value` you added. Makes sense @surbhigoel191 ?

Reply:
@isabelcosta Got it! Can I make another commit in the same PR?

Reply:
yes sure :D as many as you need @surbhigoel191